statusCodes.js

```diff
@@ -0,0 +1,14 @@
+module.exports = {
+   errorundefined: {
```

Comment:
variable naming - JavaScript devs use "camelCase" as a convention for how they write variable names. So the first letter in a variable isn't capitalized (unless that variable contains a class/constructor function), but the subsequent letters at the start of each word are. 

So `errorundefined` should be `errorUndefined`